src/lib/utils/numericUuid.ts

```diff
@@ -0,0 +1,4 @@
+function numericUuid() {
+    return Math.round(Math.random() * 10000000000000000);
```

Comment:
Let's not use `Math.random()` to create random numbers, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
> Math.random() does not provide cryptographically secure random numbers. Do not use them for anything related to security. Use the Web Crypto API instead, and more precisely the window.crypto.getRandomValues() method.